### ISSUE TITLE
Braces on migrations using Laravel preset

### ISSUE THREAD

**Issue by billmn** [laravel/pint#14]
- Pint Version: 0.1.3
- PHP Version: 8.1.7

### Description:
Laravel `make:migration` command create a migration file with braces on a new line. (e.g. https://github.com/laravel/laravel/blob/9.x/database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php)
```php
return new class extends Migration
{
    // ....
};
```

Pint change this style even using Laravel preset:

```diff
-return new class extends Migration
-{
+return new class extends Migration {
    // ...
};
```

### Steps To Reproduce:
1. Create a Laravel migration using `artisan make:migration`
2. Run `/vendor/bin/pint --preset laravel`

**Comment by Jubeki** [comment]
Discussion about anonymous constructs and braces: https://github.com/matt-allan/laravel-code-style/issues/47

**Comment by nunomaduro** [comment]
@Jubeki Not sure I understood. But do you plan to tackle this one? Otherwise I can do it tomorrow.

**Comment by Jubeki** [comment]
I am sorry I don't have time until next week. I just wanted to point to a resource which already discussed this.

So it's all yours.

**Comment by nunomaduro** [comment]
Fixed on `v0.2.0`.
